esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschuetzer2016.py

```diff
@@ -0,0 +1,98 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer script.
+
+# #############################################################################
+  Landschuetzer 2016
+# #############################################################################
+
+  Tier
+     Tier 2: other freely-available dataset.
+
+  Source
+     https://www.nodc.noaa.gov/archive/arc0105/0160558/3.3/data/0-data/
+
+  Last access
+     20190308
+
+  Download and processing instructions
+     Download the following file in ${RAWOBS}/Tier2/Landschuetzer2016:
+      spco2_1982-2015_MPI_SOM-FFN_v2016.nc
+
+  Modification history
+     20190227-A_lova_to: written.
+
+# #############################################################################
+
+"""
+
+import logging
+import os
+
+import iris
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _fix_var_metadata,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+CFG = _read_cmor_config('Landschuetzer2016.yml')
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    # fix for bad missing value definition
+    cube.data = np.ma.masked_values(cube.data, cube.data.fill_value)
```

Comment:
I doubt whether this actually does anything

Reply:
we already discussed about this with @valeriupredoi twicein this PR ... see https://github.com/ESMValGroup/ESMValTool/pull/942#issuecomment-476629690

Reply:
I recap here for brevity: raw data have weird fillvalue attributes and iris when loading the cube throws the following 
`WARNING: missing_value not used since it cannot be safely cast to variable data type`
so it loads the missing_value in the cube, but the mask is set to false.
In cube's data values contains the correct missing_value but the mask has to be regenerated and at netcdf save point I also force iris to use the desired `FillValue`.

Reply:
Ok, in that case I would recommend also doing that in a lazy way, i.e.
```python
from dask import array as da
cube = cube.copy(da.ma.masked_equal(cube.core_data(), 1e20)) 
```

Reply:
what about using the cube fill_value that is correct instead of hardcoding it here ...
```python
    fillval = cube.data.fill_value
    cube = cube.copy(da.ma.masked_equal(cube.core_data(), fillval))
```

Reply:
Calling cube.data without assigning to it realizes the data, so that defeats the purpose of keeping the data lazy.

You could use a callback when loading:
```python
def fix_fillvalue(cube, field, filename): 
    if hasattr(field.cf_data, 'missing_value'): 
        cube.data = da.ma.masked_equal(cube.core_data(), field.cf_data.missing_value)

cubes = iris.load(raw_info['file'], callback=fix_fillvalue)
```

Reply:
that's a way better. I put it in the last commit.